ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/ethernet_interfaces.py

```diff
@@ -32,6 +32,7 @@ def ethernet_interfaces(self) -> list | None:
                 "description": self.shared_utils.interface_descriptions.underlay_ethernet_interfaces(link["type"], link["peer"], link["peer_interface"]),
                 "speed": link.get("speed"),
                 "shutdown": self.shared_utils.shutdown_interfaces_towards_undeployed_peers and not link["peer_is_deployed"],
+                "sflow": link.get("sflow"),
```

Comment:
I think you only want it on the ethernet interface in the underlay_p2p case, so move this to the section below. For port-channel uplinks we should put it on the port-channel instead.

Reply:
This should be moved to the update below, so insert after line 47.